src/white_point.rs

```diff
@@ -0,0 +1,147 @@
+use num::Float;
+
+use {Xyz, flt};
+
+///WhitePoint defines the Yxy color co-ordinates for a given white point.
+///
+///A white point (often referred to as reference white or target white in technical documents)
+///is a set of tristimulus values or chromaticity coordinates that serve to define the color
+///"white" in image capture, encoding, or reproduction.
+pub trait WhitePoint<T: Float>: Sized {
+    ///Get the Yxy chromacity co-ordinates for the white point.
+    fn get_xyz() -> Xyz<Self, T>;
+}
+
+///CIE standard illuminant A is intended to represent typical, domestic, tungsten-filament lighting.
```

Comment:
Would you mind adding a short "header" or "summary" paragraph to these? Something like this should do:

```rust
///CIE standard illuminant A.
///
///...
```

It will look much nicer in the docs.